openvdb/INSTALL

```diff
@@ -52,9 +51,9 @@ Optional:
 
 - OpenGL 3.2 or later, for the OpenVDB viewer
 
-- Python 2.5, 2.6 or 2.7, for the Python module
+- Python 2.7, for the Python module
```

Comment:
This can be "2.7 or later", since we support at least some versions of Python 3.

Reply:
I thought the point in this file was to only include what we explicitly test for:

`Other versions might work but have not been tested.`

Nick fixed an issue in Python 3 recently (#428) that was reported to us - https://github.com/AcademySoftwareFoundation/openvdb/issues/427. I'm not sure we should include anything here that we don't know is actively being tested, ideally through CI?

Reply:
Perhaps, but then we should really drop all of the "or later" qualifiers.

We've supported Python 3 for some time now (see, e.g., PR #78 and the Fedora [openvdb-python3](https://apps.fedoraproject.org/packages/openvdb-python3) package).